.eslintignore

```diff
@@ -0,0 +1 @@
+build/
```

Comment:
[ESLint](https://eslint.org/) is what's used find and fix problems in JavaScript code. This file tells ESLint to ignore the `build/`, which is generated when `yarn build` is run.